monitoring/dashboards/eigenda-metrics.json

```diff
@@ -572,6 +572,92 @@
       "title": "Batch by status",
       "type": "stat"
     },
+{
+      "datasource": {
+        "type": "prometheus",
+        "uid": "prometheus"
+      },
+      "description": "",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "noValue": "Reachability metrics not found",
```

Comment:
I think it should say `not found` because this can occur when:
1. operator upgraded to 0.7.1 without adding `NODE_DATAAPI_URL` to their `.env`
2. data api is down or throwing 500 errors (not operator fault)
3. operator node is not registered yet (we only port check registered nodes because we pull socket conf onchain)

Reply:
what's the action items for operator when it's `not found` ?

Reply:
Updated to `Reachability metrics not found. Check logs for errors`